srlife/damage.py

```diff
@@ -121,11 +123,86 @@ def calculate_element_log_reliability(self, mandel_stress, temperatures, volumes
 
     svals = material.strength(temperatures)
     mvals = material.modulus(temperatures)
-    
     kvals = svals**(-mvals)
-    
+
     return -kvals * np.sum(pstress**mvals[...,None], axis = -1) * volumes
 
+class WeibullNormalTensileAveragingModel(WeibullFailureModel):
+  """
+    Weibull normal tensile average failure model
+
+    Assigning default values for nalpha and nbeta
+  """
+
+  def __init__(self, pset, *args, **kwargs):
+    super().__init__(pset, *args, **kwargs)
+    # limits and number of segments for angles
+    self.alphai = pset.get_default("alphai",0)
+    self.alphaf = pset.get_default("alphaf",np.pi)
+    self.betai = pset.get_default("betai",0)
+    self.betaf = pset.get_default("betaf",2*np.pi)
```

Comment:
We can and should hard code the angle limits, there is no reason why the user should have need to change those.  So something like

```
self.betaf = 2*np.pi
```

here is fine.

Reply:
Sure we can hard code them and get rid of the extra variables.